### DISCUSSION TITLE
How do I change a task to open the dock on the right and/or focused?

### DISCUSSION THREAD

**Discussion by smpallen99** [zed-industries/zed#8481] [Feature Requests]
I would like to do 2 things.

* change the terminal settings to open the doc on the right, instead of the bottom
* configure (an exsiting task) to open the doc in focused mode

For the latter, I'm used a task for lazygit, like I do with neovim.

**Comment by Moshyfawn** [comment]
For dock placement, update `terminal.dock'.

```json
"terminal": {
  "dock": "right"
}
```

In terms of opening the dock in focused mode, I think [you've already noticed](#8485) that there's a limitation at the moment with the initial task support, where you can't run an async task and then send a command at this time.

**Reply by smpallen99** [reply]
Thanks, I searched the docs for a while and could not find that option. That's the issue with the current documentation. It only covers the basics for getting started. And I'm not familar with Rust, so reading the source is a little ovrewhelming. I'm looking forward to the day when the documentation is complete. 

Since that was a little negative, let me end with "I love the editor, even with all its feature gaps in the present state." Can't wait until the vim bindings are more complete or we get the abilitiy to extend it

**Reply by Moshyfawn** [reply]
I hear you on the "basic documentation" point. There are quite a few parts of Zed that are in flux right now, especially after the underlining GPUI rewrite. But don't worry, we're working on figuring out the best way to document everything and it should get better over time 💯

**Reply by smpallen99** [reply]
Yes, I'm sure they'll get done. I'm really impressed with the response time I'm getting from your team. Great Job.
